package.json

```diff
@@ -29,6 +29,7 @@
     "clean": "npm run clean:build",
     "clean:docs": "rimraf docs/api/*",
     "clean:build": "rimraf build",
+    "flow": "flow; test $? -eq 0 -o $? -eq 2",
```

Comment:
why not to use `flow check` instead `flow`? and you don't need to check test exit status in this case.